backend/src/db/migrations/20230923060705_add_surveys_steps_table.ts

```diff
@@ -0,0 +1,36 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'surveys';
+
+const ColumnName = {
+  FEELINGS: 'feelings',
+  GOALS: 'goals',
+  WORRIES: 'worries',
+  MEDITATION_EXPERIENCE: 'meditation_experience',
+  JOURNALING_EXPERIENCE: 'journaling_experience',
+} as const;
+
+const POSTGRE_ARRAY_TYPE = 'varchar(1000)[]';
+const POSTGRE_STRING_TYPE = 'varchar(255)';
```

Comment:
As I understand, default `table.string('smth')` already creates varchar(255), so there's no reason to specify type this way

Reply:
done